server/service/src/vaccination/update/mod.rs

```diff
@@ -501,6 +501,14 @@ mod update {
         // ----------------------------
         // Update: Not given -> given, from another store
         // ----------------------------
+        let vaccinations_repo = VaccinationRowRepository::new(&context.connection);
```

Comment:
These two tests weren't actually testing not given -> given, so have fixed that... though they should still pass now anyways even if you do given -> given